worksheets_test.go

```diff
@@ -47,6 +47,43 @@ func (s *Zuite) TestExample() {
 	require.Equal(s.T(), false, isSet)
 }
 
+func (s *Zuite) TestWorksheetNew_zeroDefs() {
+	wsDefs := []string{
+		``,
+		`some text`,
+		`not a worksheet`,
+		`work sheet`,
+	}
+	for _, def := range wsDefs {
+		_, err := NewDefinitions(strings.NewReader(def))
+		require.Error(s.T(), err)
+	}
+}
+
+func (s *Zuite) TestWorksheetNew_multipleDefs() {
+	wsDefs := `worksheet one {1:name text} worksheet two {1:occupation text}`
+	defs, err := NewDefinitions(strings.NewReader(wsDefs))
```

Comment:
What about simplifying to simply check that `defs.defs` is `len` of `2` and has `one` and `two` as keys?